File: app/src/main/java/fr/paug/androidmakers/util/CustomTabUtil.java
package fr.paug.androidmakers.util;

import android.content.Context;
import android.net.Uri;

import androidx.browser.customtabs.CustomTabsIntent;
import androidx.core.content.ContextCompat;

import fr.paug.androidmakers.R;

public final class CustomTabUtil {

    private CustomTabUtil() {
        // no instance
    }

    public static void openChromeTab(Context context, String url) {
        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
        builder.setToolbarColor(ContextCompat.getColor(context, R.color.colorPrimary));
        CustomTabsIntent customTabsIntent = builder.build();
        customTabsIntent.launchUrl(context, Uri.parse(url));
    }

}

File: app/src/main/java/fr/paug/androidmakers/util/CustomTabUtil.kt
package fr.paug.androidmakers.util

import android.content.Context
import android.net.Uri
import androidx.browser.customtabs.CustomTabsIntent
import androidx.core.content.ContextCompat
import fr.paug.androidmakers.R

object CustomTabUtil {
    fun openChromeTab(context: Context?, url: String?) {
        val builder = CustomTabsIntent.Builder()
        builder.setToolbarColor(ContextCompat.getColor(context!!, R.color.colorPrimary))
        val customTabsIntent = builder.build()
        customTabsIntent.launchUrl(context, Uri.parse(url))
    }
}